saleor/dashboard/category/forms.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from mptt.forms import TreeNodeChoiceField
+
+from ...product.models import Category
+
+
+class CategoryForm(forms.ModelForm):
+    class Meta:
+        model = Category
+        exclude = []
+
+    def __init__(self, *args, **kwargs):
+        super(CategoryForm, self).__init__(*args, **kwargs)
+        self.fields['parent'] = TreeNodeChoiceField(queryset=Category.objects.all(), required=False)
```

Comment:
If the field does not depend on the current instance, why is it created in `__init__`?